build

```diff
@@ -3,14 +3,29 @@
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/etcd"
 
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+LINK_OPERATOR="="
+
+: ${ARCH_LIST:="amd64 arm64"}
```

Comment:
Yes. I don't think cross-compiling / building all architectures is the typical use case. Do many projects build for all archs in one shot by default?